homeassistant/components/flux_led/light.py

```diff
@@ -181,7 +181,7 @@ async def async_setup_entry(hass, entry, async_add_entities):
     if config_type == "auto":
         # Find the bulbs on the LAN
         scanner = BulbScanner()
-        scanner.scan(timeout=10)
+        await hass.async_add_executor_job(scanner.scan(timeout=10))
```

Comment:
Keyword arguments are not supported with this helper. Also we can't call the function before passing it to the helper. We should just pass the function instance and its arguments separately.

If we need to pass keyword arguments we can use `functools.partial`.

Reply:
```suggestion
        await hass.async_add_executor_job(partial(scanner.scan, timeout=10))
```

Reply:
The keyword arguments are not necessary - there is a default included in the library. In the updatecoordinator I remove all of these. I have removed the argument in this PR.